server/resp/src/main/java/org/infinispan/server/resp/filter/EventListenerConverter.java

```diff
@@ -0,0 +1,58 @@
+package org.infinispan.server.resp.filter;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Collections;
+import java.util.Set;
+
+import org.infinispan.commons.dataconversion.MediaType;
+import org.infinispan.commons.marshall.AbstractExternalizer;
+import org.infinispan.commons.marshall.AdvancedExternalizer;
+import org.infinispan.encoding.DataConversion;
+import org.infinispan.metadata.Metadata;
+import org.infinispan.notifications.cachelistener.filter.CacheEventConverter;
+import org.infinispan.notifications.cachelistener.filter.EventType;
+import org.infinispan.server.resp.ExternalizerIds;
+
+public class EventListenerConverter<K, V, C> implements CacheEventConverter<K, V, C> {
+   public static AdvancedExternalizer<EventListenerConverter> EXTERNALIZER = new EventListenerConverter.Externalizer();
+   private DataConversion dc;
```

Comment:
Although this works, the `DataConversion` needs to be initialized to handle the parsing :(. Looking at the `CacheNotifierImpl` (https://github.com/infinispan/infinispan/blob/main/core/src/main/java/org/infinispan/notifications/cachelistener/CacheNotifierImpl.java#L1605-L1619), unsure if just having a method here annotated with `@Inject` would do the trick?